### DISCUSSION TITLE
Отправка E-mail ⚡️ Node.js с примерами кода

### DISCUSSION THREAD

**Discussion by giscus** [xsltdev/nodejsdev.ru#47] [Announcements]
# Отправка E-mail ⚡️ Node.js с примерами кода

Справочник Node.js с примерами кода

https://nodejsdev.ru/guides/webdraftt/email/

**Comment by amir248** [comment]
Скопировал, а ничего не работает. (

**Reply by bndby** [reply]
Просто скопировать недостаточно, нужно под свой почтовый сервер настроить, да и в логах наверняка есть запись, в чем ошибка.

**Reply by Aleksandra-ui** [reply]
не работает потому что гугл больше не поддерживает авторизацию в сторонних приложениях

**Comment by Bahurtsev** [comment]
Если в сумме с этим видосом, то у меня получилось. https://www.youtube.com/watch?v=D1IatZ79wbI&ab_channel=webdive Предлагаю данный пример к нашему русскому майлу прикрутить.

**Reply by Bahurtsev** [reply]
А всё заработало и как здесь тоже. Дело в том что я сижу на Ютубе и получаю там пароль для почтового клиента. Но как оказалось там почта одна, а gmail почта другая. То есть я вышел и Ютуба где логин был обычный mail и зашёл ровно там же но только в gmail почту. Они буквально одинаковы вот я и перепутал.
